### DISCUSSION TITLE
Mosquitto and NPM

### DISCUSSION THREAD

**Discussion by danyo2008** [NginxProxyManager/nginx-proxy-manager#2050] [General]
Hi,

playing around with NPM since a few days. HAve used Traefik before, but this is a little bit to overloaded for me for my homelab.
What i want to know is, if it is possible to connect to a MQTT Server (modquitto) throug NPM with TLS (SSL Certs) enabled. 
I know, there is an option for Websockets, and the stream option in NPM, but with streams, it is not possible to use my local domain name with my letsencrypt certificate.

Is it somehow possible, to do that with Nginx Proxy Manager?

Would be nice, to get some answers.....

Thanks and regards
danyo

**Comment by rasulkireev** [comment]
Struggling with this too.

I have a mongodb instance working via a stream, but mqtt fails to work for some reason.
